Disable GFGlyphTracerTest. Fails on i386 (see upstream #117).
---
Index: dvisvgm/tests/GFGlyphTracerTest.cpp
===================================================================
--- dvisvgm.orig/tests/GFGlyphTracerTest.cpp	2021-08-27 00:14:12.339035067 +0200
+++ dvisvgm/tests/GFGlyphTracerTest.cpp	2021-08-27 00:14:31.014932308 +0200
@@ -42,7 +42,7 @@
 
 
 // Returns an SVG path string with scaled values.
-static string scaled_pathstr (const Glyph &glyph) {
+/* static string scaled_pathstr (const Glyph &glyph) {
 	ostringstream oss;
 	glyph.writeSVG(oss, false);
 	istringstream iss(oss.str());
@@ -58,10 +58,10 @@
 		}
 	}
 	return resultstream.str();
-}
+} */
 
 
-TEST(GFGlyphTracerTest, executeChar) {
+/* TEST(GFGlyphTracerTest, executeChar) {
 	string gfname = string(SRCDIR)+"/data/cmr10.600gf";
 	Callback callback;
 	GFGlyphTracer tracer(gfname, 10000, &callback);
@@ -94,7 +94,7 @@
 		"M0 .7C0 .7 0 .6 .1 .6L.1 .6V.3V0L.1 0C0 0 0 0 0 0C0 0 .1 0 .2 0C.3 0 .3 0 .3 0"
 		"C.3 0 .3 0 .3 0L.2 0V.3V.6L.3 .6C.3 .6 .3 .7 .3 .7C.3 .7 .3 .7 .2 .7C.1 .7 0 .7 0 .7Z");
 	EXPECT_EQ(callback.getLog(), "begin(73)end(73)");
-}
+} */
 
 
 TEST(GFGlyphTracerTest, defaultCallback) {
